Sources/SQLKit/Query/SQLUpdate.swift

```diff
@@ -27,5 +29,6 @@ public struct SQLUpdate: SQLExpression {
             serializer.write(" WHERE ")
             predicate.serialize(to: &serializer)
         }
+        returning?.serialize(to: &serializer)
```

Comment:
If this is not nil, a leading space should be serialized.

Reply:
Note that SQLSerializer's statement helper does this for you:
```swift
// ALTER TYPE enum_type ADD VALUE 'new_value';
public struct SQLAlterEnum: SQLExpression {
    public var name: SQLExpression
    public var value: SQLExpression?

    public func serialize(to serializer: inout SQLSerializer) {
        serializer.statement {
            $0.append("ALTER TYPE")
            $0.append(self.name)
            if let value = self.value {
                $0.append("ADD VALUE")
                $0.append(value)
            }
        }
    }
}
```
I've been upgrading things to use this pattern as I go.

Reply:
I like the statement helper!